### ISSUE TITLE
Error de acciones de comando sin existir 

### ISSUE THREAD

**Issue by 0372JesusMtz0372** [Ajneb97/EasyCommandBlocker#12]
Hola Ajneb

Al usar un comando que no existe por ejemplo `/ajneb` se ejecutan las acciones de `blocked_command_default_actions:`, yo esperaría que tenga diferentes acciones algo como`not_exist_command_actions:` y que estas acciones se ejecuten cuando un comando no existe, tal vez también puedes agregar cuando un comando si existe pero este está incorrecto.

**Comment by 0372JesusMtz0372** [comment]
Tal vez una solución sería que primero compruebe si existe el comando en caso que no ejecute las acciones correspondientes y si existe compruebe si está bloqueado

**Comment by GreatGoal0** [comment]
Si usa un comando que no existe ``/ajneb`` le salga un mensaje
>El comando no existe o está incorrecta
 
Si usa un comando que si existe ``/version`` pero este bloqueado le salga un mensaje
>No tienes permisos para usar este comando
 
Si usa un comando que si existe y no está bloqueado pues normal puede ejecuta el comando

**Comment by Ajneb97** [comment]
Nunca lo pense de esa forma, si agrego una sección como la que pides, el funcionamiento del plugin cambiará completamente para los que ya lo usan. Tendria que pensarmela para ver como no perjudicar a la configuración actual.

**Comment by GreatGoal0** [comment]
No es posible que la opción sea opcional y que las personas lo agreguen manualmente para que no se requiera y afecte la config

**Comment by GreatGoal0** [comment]
Si agregas una placeholder en ConditionalEvents que compruebe si el comando que uso el usuario es válido y así CE envié el mensaje correcto

**Comment by GreatGoal0** [comment]
Hubo solución?

**Comment by GreatGoal0** [comment]
Se agrego la función?

**Comment by Ajneb97** [comment]
No, no he tenido nada de tiempo, ademas estoy dedicandome a mis otros plugins mas recientes y premium.
